package.json

```diff
@@ -112,7 +113,7 @@
   },
   "peerDependencies": {
     "@adonisjs/assembler": "^7.0.0",
-    "@adonisjs/core": "^6.2.0",
```

Comment:
Safely bumped peerDependency version for adonisjs/core since we use @adonisjs/core/test_utils submodule available since 6.2.2